addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I prefer multiple lines because:
1. It is easy to be consistent on when to break the line. When is it too long? If there is more than x split onto a new line.
2. If I am working on something and I want to change what is destructured I can just comment out that line.
3. To me it is easier to read.